messaging/src/main/java/com/mx/path/connect/messaging/remote/RemoteCRUDModel.java

```diff
@@ -128,7 +129,7 @@ public T create(T obj) {
     MessageResponse response = executeRequest(clientId, request);
 
     if (response.getStatus() != MessageStatus.SUCCESS) {
-      throw new MessageError("Unable to create remote " + RemoteChannel.getModel(getClassOfT()) + " with status " + response.getStatus(), response.getStatus(), response.getException());
+      throw new FacilityException("Unable to create remote " + RemoteChannel.getModel(getClassOfT()) + " with status " + response.getStatus(), response.getException());
```

Comment:
It is preferred that you create a custom exception that is more descriptive of what is happening and throw that. Don't we already have a configuration exception that descends from FacilityException?

Reply:
@mattnichols  Created a custom exception class named MessageBrokerException.